TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robotSubSystems/SubSystemManager.java

```diff
@@ -29,14 +29,17 @@ public class SubSystemManager {
     private static IntakeState intakeState = IntakeState.STOP;
     private static boolean lastRightBumperButtonState;
     private static boolean lastYButtonState;
+    private static boolean lastxButtonState;
     private static boolean rightBumperControl = false; // TODO maybe a better name?
     private static boolean yButtonControl = false; // TODO maybe a better name?
     private static boolean rightStickControl = false; // TODO maybe better name?
+    private static boolean lastLeftBumperButtonState;
+    private static boolean depleteControl = true;
 
     private static RobotState getState(Gamepad gamepad) {
         return gamepad.b ? RobotState.TRAVEL
                 : gamepad.a ? RobotState.INTAKE
-                        : gamepad.x ? RobotState.DEPLETE : gamepad.left_bumper ? RobotState.CLAWINTAKE : null;
+                        : gamepad.right_bumper ? RobotState.DEPLETE : gamepad.dpad_up ? RobotState.CLAWINTAKE : null;
```

Comment:
Why right bumper?

Reply:
thats what we talked about. Whenever shemi prreses the right bumber button the elavator returns to the position from the second driver. We chose this button because he is relatively close to the triangle button